### PR TITLE
Added Landing page

### PR DESCRIPTION
**Pull request by Shah-Arsalan**

### CODE REVIEW

**Review by Rohanmond — APPROVED** [review]

**Review by hrishi82 — APPROVED** [review]
Looks good!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        </div>
+
+        <div class="icons">
+            <a href=""><button class="primary-button">Login</button></a>
+        </div>
+
+    </div>
+
+    <div class="image-container-home">
+        <img src="/images/Saly-15 (1).png" alt="">
```
Comment (anuj-kumary) [inline index.html:27]: give value to **alt**

File: index.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+font-family: var(--default-font-family);
+  }
+
+
+  .nav-container{
+      padding: 0;
+      background-color:#0c0d13 ;
+      height: 100px;
+  }
+
```
Comment (anuj-kumary) [inline index.css]: Don't hard code color value use as variable
Do format document so it looks NYC

File: index.html (hunk 1 above)
Comment (Shah-Arsalan) [reply index.html:27]: okay will do

**Review by Sharmaa-Ansh — APPROVED** [review]

File: index.css (hunk 2 above)
Comment (Shah-Arsalan) [reply index.css]: okay changing

**Pull request merged**